Gruntfile.js

```diff
@@ -364,16 +293,6 @@ module.exports = function (grunt) {
         },
         keepalive: true
       },
-      server: {
-        options: {
-          open: false
-        }
-      },
-      open: {
-        options: {
-          open: true
```

Comment:
I personally like the auto-open behavior - is there a reason why we decided to remove it?

Reply:
hah, I guess it's more personal preference whether turn on auto-open.
will enable it in the webpack-dev-server config